samcli/lib/remote_invoke/remote_invoke_executor_factory.py

```diff
@@ -127,6 +129,41 @@ def _create_lambda_boto_executor(
             log_consumer=log_consumer,
         )
 
+    def _create_stepfunctions_boto_executor(
+        self,
+        cfn_resource_summary: CloudFormationResourceSummary,
+        remote_invoke_output_format: RemoteInvokeOutputFormat,
+        response_consumer: RemoteInvokeConsumer[RemoteInvokeResponse],
+        log_consumer: RemoteInvokeConsumer[RemoteInvokeLogOutput],
+    ):
+        """Creates a remote invoke executor for Step Functions resource type based on
+        the boto action being called.
+
+        :param cfn_resource_summary: Information about the Lambda resource
+        :param remote_invoke_output_format: Response output format
+        :param response_consumer: Consumer for writing responses to stdout
+        :param log_consumer: Log consumer for writing logs to stderr
+
+        :return: Returns the created remote invoke Executor
```

Comment:
We use nupy style docstrings (I think we missed some other places in this file)

https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#code-conventions

We should be having that as a style check, until then can you update docstrings in the changed files to align with our dev-guide?